sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -39,10 +39,15 @@
       <artifactId>azure-core</artifactId>
       <version>1.10.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-experimental</artifactId>
+      <version>1.0.0-beta.8</version> <!-- {x-version-update;com.azure:azure-core-experimental;dependency} -->
+    </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
+      <version>1.7.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->
```

Comment:
This should use `beta_com` for 1.7.0-beta.2 for the latest azure-core-amqp beta release.